microsoft-edge/webview2/releasenotes.md

```diff
@@ -20,6 +20,32 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
 > [!IMPORTANT]
 > While WebView2 is a preview, the .NET APIs are in the `pre-release package`.  
 
+## 0.9.579  
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  
+
+
+#### General  
+
+*   > [!IMPORTANT]
+    > **Announcement**:  Support for WebView2 SDK Versions [0.8.190](#08190), [0.8.230](#08230), [0.8.270](#08270), [0.8.314](#08314), and [0.8.355](#08355) will be dropped after the next SDK release. These versions are also marked deprecated on nuget.org. The WebView team recommends staying up to date with the latest version of WebView2.
+
+*   Added ICoreWebView2EnvironmentOptions::IsSingleSignOnUsingOSPrimaryAccountEnabled to enable conditional access for WebView.
+*   Added WebResourceResponseReceived event API for authentication request and include authentication headers.
```

Comment:
Recommend something like `Added WebResourceResponseReceived event that is fired when WebView2 receives HTTP responses. Authentication headers, if any, are included in the response object.`

Also, should link to the API/event in reference doc.